applications/GeoMechanicsApplication/custom_utilities/stress_strain_utilities.h

```diff
@@ -39,9 +42,25 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) StressStrainUtilities
                                                 bool                       UseHenckyStrain,
                                                 std::size_t                VoigtSize);
 
+    static constexpr std::size_t GetVoigtSize(std::size_t Dimension)
+    {
+        return Dimension == N_DIM_3D ? VOIGT_SIZE_3D : VOIGT_SIZE_2D_PLANE_STRAIN;
+    }
+
+    static constexpr std::size_t GetStressTensorSize(std::size_t Dimension)
+    {
+        return Dimension == N_DIM_3D ? STRESS_TENSOR_SIZE_3D : STRESS_TENSOR_SIZE_2D;
```

Comment:
This is of course the functionality that was already there and this comment is not something we need to fix in this PR I think, but do you think we should also be able (at some point) to handle a Dimension of 1?